components/dashboard/src/settings/selectClass.tsx

```diff
@@ -48,24 +58,19 @@ export default function SelectWorkspaceClass(props: SelectWorkspaceClassProps) {
                     Choose the workspace machine type for your workspaces.
                 </p>
                 <div className="mt-4 space-x-3 flex">
-                    <WorkspaceClass
-                        additionalStyles="w-80 h-32"
-                        selected={workspaceClass === "g1-standard"}
-                        onClick={() => actuallySetWorkspaceClass("g1-standard")}
-                        category="GENERAL PURPOSE"
-                        friendlyName="Standard"
-                        description="Up to 4 vCPU, 8GB memory, 30GB disk"
-                        powerUps={1}
-                    />
-                    <WorkspaceClass
-                        additionalStyles="w-80 h-32"
-                        selected={workspaceClass === "g1-large"}
-                        onClick={() => actuallySetWorkspaceClass("g1-large")}
-                        category="GENERAL PURPOSE"
-                        friendlyName="Large"
-                        description="Up to 8 vCPU, 16GB memory, 50GB disk"
-                        powerUps={2}
-                    />
+                    {supportedClasses.map((c) => {
```

Comment:
Nice! Now it becomes apparent that this config is far to hidden. But now we have everything in place to maybe show it on `/workspaces` / the "open workspace modal" / somewhere else for visible? :innocent:  

:cloud: Clearly out-of-scope here, just wanted to raise in this context.

/cc @gtsiolis